src/guides/v2.4/release-notes/release-notes-2-4-0-commerce.md

```diff
@@ -826,7 +921,7 @@ We have fixed hundreds of issues in the Magento 2.4.0 core code.
 
 <!--- MC--->
 
-*  The integration of third-party payment methods into the core Magento code has been deprecated. With this release, the integrations of the Authorize.Net, eWay, CyberSource, and Worldpay payment methods are deprecated. These core features are no longer supported and will be removed in the next minor release (2.4.0). Merchants should migrate to the official extensions that are available on the Magento Marketplace.
+*  The integration of third-party payment methods (Authorize.Net, eWay, CyberSource, and Worldpay) into the core Magento code has been deprecated. Merchants should migrate to the official extensions that are available on the Magento Marketplace.
```

Comment:
These have been removed now. Also Braintree

Authorize.Net and Braintree are in CE, so a modified version release note needs to be in the Open Source relnotes.

Reply:
entered edit but kept all payment methods in OS notes -- PM says these are all open source.